go.mod

```diff
@@ -2,27 +2,27 @@ module github.com/AVOXI/ari-proxy/v5
 
 go 1.14
 
-replace github.com/CyCoreSystems/ari/v5 => github.com/AVOXI/ari/v5 v5.0.7-0.20210406164555-0fd81698ba3e
+replace github.com/CyCoreSystems/ari/v5 => github.com/AVOXI/ari/v5 v5.0.7-0.20220105030036-1b48da28b0de
```

Comment:
this is pointing at the v5-gogo-free branch, right?

Reply:
Yes. It was so tricky to figure that what this was supposed to point it. :P